services/agents/conn.go

```diff
@@ -29,16 +28,21 @@ import (
 )
 
 type Conn struct {
-	stream agent.Agent_ConnectServer
-	lastID uint32
-	l      *logrus.Entry
+	stream            agent.Agent_ConnectServer
+	lastID            uint32
+	l                 *logrus.Entry
+	messageDispatcher *messageDispatcher
 }
 
 func NewConn(uuid string, stream agent.Agent_ConnectServer) *Conn {
-	return &Conn{
-		stream: stream,
-		l:      logrus.WithField("pmm-agent", uuid),
+	// Create goroutine to dispatch messages
```

Comment:
We have two common styles for internal (i.e. not for method documentation, etc.) commands:
1.
```
sentence fragment, starting with a lower-case letter and not ending with a dot
```
2.
```
Several full sentences. Each starts with an upper-case letter. Each ends with a dot.
```

Please use the first one there.